js/cli/src/namespace.ts

```diff
@@ -1,39 +1,44 @@
 #!/usr/bin/env ts-node
-import { Wallet, CLI } from "@raindrop-studios/sol-command";
-import { AnchorProvider, web3 } from "@project-serum/anchor";
+import { CLI } from "@raindrop-studios/sol-command";
+import * as anchor from "@project-serum/anchor";
+import { web3 } from "@project-serum/anchor";
 import log from "loglevel";
-
-import { NamespaceProgram } from "@raindrops-protocol/raindrops";
+import {
+  IDL as NamespaceProgramIDL,
+} from "../../idl/namespace";
+import { NamespaceProgram } from "../../lib/src/contract/namespace";
+import * as nsIx from "../../lib/src/instructions/namespace";
+import fs from "fs";
```

Comment:
Convention we've been using is to have system/external modules imported above local modules. Move `fs` import above line 6.